src/framework/components/camera/component.js

```diff
@@ -289,6 +289,14 @@ class CameraComponent extends Component {
         }
     }
 
+    set renderSceneColorMap(value) {
+        if (value) {
+            this._colorRequestedBefore = true;
```

Comment:
please add this to constructor as well, so that we do not introduce class members during runtime.